src/main/java/com/lohika/morning/ecs/menu/ManagerMenu.java

```diff
@@ -14,10 +15,12 @@ public ManagerMenu(UI ui, List<MorningEvent> events) {
     super(ui);
     addComponent(createNavigationButton("Events", EventListView.VIEW_NAME));
 
-    events.forEach(morningEvent -> {
-      String viewName = EventDetailsView.VIEW_NAME + "/" + morningEvent.getEventNumber();
-      Button button = createNavigationButton(morningEvent.getName(), viewName);
-      addComponent(button);
-    });
+    events.stream()
+        .sorted(Comparator.comparing(MorningEvent::getEventNumber))
```

Comment:
Do we need to filter out non-editable events?

Reply:
yes we need) but not in this PR